catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/impl/operations/OverrideAttributesSupportTest.java

```diff
@@ -63,7 +63,7 @@ public void testOverrideAttributesBasic() throws URISyntaxException {
                 .getTitle(), is("updated"));
         assertThat(metacardMap.get("original")
                 .getResourceURI()
-                .toString(), is("content:stuff"));
```

Comment:
I understand it has no impact on the test results, but it is confusing/misleading to anyone who tries to look at our unit tests for examples of how the software should be used or should behave. 😺